.github/workflows/cleanup.yml

```diff
@@ -12,11 +12,19 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 5
     steps:
-      - name: 'Cancel Runs For Closed PRs'
+      - name: 'Cancel CI Runs For Closed PRs'
         uses: styfle/cancel-workflow-action@0.9.0
         with:
           # Cancel workflow when PR closed. https://github.com/styfle/cancel-workflow-action#advanced-ignore-sha
           ignore_sha: true
           # Note: workflow_id can be a Workflow ID (number) or Workflow File Name (string).
           workflow_id: "ci.yml"
           access_token: ${{ github.token }}
+      - name: 'Cancel Docs Runs For Closed PRs'
+        uses: styfle/cancel-workflow-action@0.9.0
+        with:
+          # Cancel workflow when PR closed. https://github.com/styfle/cancel-workflow-action#advanced-ignore-sha
+          ignore_sha: true
+          # Note: workflow_id can be a Workflow ID (number) or Workflow File Name (string).
+          workflow_id: "docs.yml"
+          access_token: ${{ github.token }}
```

Comment:
Can we use a comma separated list instead? Please ignore if you already considered it. 

https://github.com/styfle/cancel-workflow-action#advanced-canceling-other-workflows
> Note: workflow_id also accepts a comma separated list if you need to cancel multiple workflows

Reply:
Ah, I was looking for the array syntax but didn't find it. Thank you.